bears/c_languages/CPPLintBear.py

```diff
@@ -26,12 +32,20 @@ class CPPLintBear:
     def create_arguments(filename, file, config_file,
                          max_line_length: int=79,
                          cpplint_ignore: typed_list(str)=(),
-                         cpplint_include: typed_list(str)=()):
+                         cpplint_include: typed_list(str)=(),
+                         indent_size: int=2,
+                         use_spaces: bool=True,):
         """
         :param max_line_length: Maximum number of characters for a line.
         :param cpplint_ignore:  List of checkers to ignore.
         :param cpplint_include: List of checkers to explicitly enable.
+        :param use_spaces:      True if spaces are to be used instead.
```

Comment:
This will be used in documentation.  It should say that the only acceptable value is `True`.  Likewise for `indent_size`.